core/src/main/java/org/testng/internal/invokers/ParameterHandler.java

```diff
@@ -74,9 +76,11 @@ private ParameterBag handleParameters(
               holder);
       return new ParameterBag(paramHolder);
     } catch (Throwable cause) {
-      String msg = Utils.longStackTrace(cause.getCause() != null ? cause.getCause() : cause, true);
-      if (Strings.isNotNullAndNotEmpty(msg)) {
-        Utils.error(msg);
+      if (verbose >= 2) {
```

Comment:
Why not having `Utils. getVerboseLevel()` instead?

Reply:
Why use global variables when there's an easily accessible suite-local verbosity setting?

Reply:
Because I see logging has an aspect without a string link in the business logic.

But tbh, this change was at the top of the PR and I supposed it impacted more changes.